app/models/health_care_application.rb

```diff
@@ -283,21 +287,23 @@ def log_submission_failure_details
   def send_failure_email
     if Flipper.enabled?(:hca_submission_failure_email_va_notify)
       begin
-        email = parsed_form['email']
+        first_name = parsed_form.dig('veteranFullName', 'first')
         template_id = Settings.vanotify.services.health_apps_1010.template_id.form1010_ez_failure_email
         api_key = Settings.vanotify.services.health_apps_1010.api_key
 
+        personalisations = first_name ? { 'salutation' => "Dear #{first_name}," } : nil
```

Comment:
I am going to need to test this in staging to confirm the behavior of not passing a personalisation here. The goal is to provide the name if we have it, but since this was from an error maybe something happened and we don't have it so I wanted to handle that scenario. The first name is required so I am hoping this does not happen.